### PR TITLE
manifest: sdk-zephyr: Update manifest to sdk-zephyr

### PR DESCRIPTION
**Pull request by ryanjh**
Add missing functionality required for ncs 1.4 qualification

Update sdk-zephyr: nrfconnect/sdk-zephyr#385

Signed-off-by: Ryan Chu <ryan.chu@nordicsemi.no>

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: v2.4.0-ncs1
+      revision: v2.4.0-ncs-branch
```
Comment (joerchan) [inline west.yml:53]: @ryanjh We don't put branch, but the current head of this branch.
So it should be: 38b88bdfa73ea632cb1cf34590c7d0c410fa6aa1

**Review by joerchan — CHANGES_REQUESTED** [review]

File: west.yml (hunk 1 above)
Comment (ryanjh) [reply west.yml]: thanks Joakim, I've fixed it.

**Review by joerchan — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]
SHA checked.

**Comment by joerchan** [comment]
Will be included by https://github.com/nrfconnect/sdk-nrf/pull/3312

CI decided to be troublesome.

**Pull request closed without merging**